invisible_cities/evm/pmaps_test.py

```diff
@@ -305,3 +315,58 @@ def test_PMap_s2s(pmps):
     assert len(pmp.s2s) == len(s2s)
     for kept_s2, true_s2 in zip(pmp.s2s, s2s):
         assert_Peak_equality(kept_s2, true_s2)
+
+
+
+@fixture(scope='module')
+def noise_func():
+    return NoiseSampler('new', 6400).signal_to_noise
+
+
+@fixture(scope='module')
+def s2_peak():
+    times      = np.arange(0, 20 * units.mus, 1 * units.mus)
+    bin_widths = np.full(len(times), 1 * units.mus)
+
+    pmt_ids  =    DB.DataPMT ('new', 6400).SensorID.values
+    n_sipms = len(DB.DataSiPM('new', 6400).SensorID.values)
+    sipm_ids = np.arange(n_sipms)
```

Comment:
```suggestion
    sipm_ids = DB.DataSiPM('new', 6400).index.values
```

removing the previous line alse

Reply:
It can be also used for PMTs, but it doesn't matter